src/skills-builder/skills-builder-modal/view-results/ViewResults.jsx

```diff
@@ -27,35 +28,12 @@ const ViewResults = () => {
   const [isLoading, setIsLoading] = useState(true);
   const [fetchError, setFetchError] = useState(false);
 
-  useEffect(() => {
-    const getRecommendations = async () => {
-      // fetch list of jobs with related skills
-      const jobInfo = await searchJobs(jobSearchIndex, careerInterests);
-
-      // fetch course recommendations based on related skills for each job
-      const results = await Promise.all(jobInfo.map(async (job) => {
-        const formattedSkills = job.skills.map(skill => skill.name);
-
-        // create a data object for each job
-        const data = {
-          id: job.id,
-          name: job.name,
-          recommendations: {},
-        };
-
-        // get recommendations for each product type based on the skills for the current job
-        await Promise.all(productTypes.map(async (productType) => {
-          const response = await getProductRecommendations(productSearchIndex, productType, formattedSkills);
+  const productTypes = useProductTypes();
 
-          // replace all white spaces with an underscore
-          const formattedProductType = productType.replace(' ', '_');
-
-          // add a new key to the recommendations object and set the value to the response
-          data.recommendations[formattedProductType] = response;
-        }));
-
-        return data;
-      }));
```

Comment:
This has been refactored into the `service.js` file. This is more in line with our best practices and also cleans up the file a little bit.